src/ui/public/collapsible_sidebar/_index.scss

```diff
@@ -0,0 +1,3 @@
+@import '../styles/variables/depth';
+
```

Comment:
no need for extra space, right?

Reply:
I tend to add spaces between imports that are just variable inputs versus component inputs.